config/locales/en/admin/change_note_form.yml

```diff
@@ -0,0 +1,11 @@
+en:
+  activemodel:
+      errors:
+        models:
+          change_note_form:
+            attributes:
+              minor_change:
+                inclusion: Select whether this is a major or minor update
+              change_note:
+                blank: "Enter a change note"
```

Comment:
This is a new pattern of adding error messages, rather than adding them to the models themselves. I know during discussions around improving errors down the line, we've discussed extracting them from the models and moving them to locales, so i figured this is probs a good place to start.

These definitely could just be added to the validations in the form object. This would def help to alleviate a bit of model bloat though,

Reply:
I'm in favour. Would also make our tests more maintainable as it would mean we could just reference the locale string instead of having to update our tests every time we tweak an error message

Reply:
Great shout. I should def be using that in the test. Will sort that now